deployment/enterprise/docker-compose.yml

```diff
@@ -0,0 +1,104 @@
+name: mergin-enterprise
+networks:
+  mergin:
+    name: mergin-ee
+
+services:
+  server:
+    image: 433835555346.dkr.ecr.eu-west-1.amazonaws.com/mergin/mergin-ee-back:2025.3.0
+    container_name: mergin-server-enterprise
+    restart: always
+    user: 901:999
+    command: ["gunicorn -w 4 --config config.py application:application"]
+    volumes:
+      - ./data:/data  # map data dir to host
+      - ./entrypoint.sh:/app/entrypoint.sh
+    env_file:
+      - .prod.env
+    depends_on:
+      - db
+    networks:
+      - mergin
+
+  web:
+    image: 433835555346.dkr.ecr.eu-west-1.amazonaws.com/mergin/mergin-ee-front:2025.3.0
+    container_name: mergin-web-enterprise
+    restart: always
+    depends_on:
+      - server
+    env_file:
+      - .prod.env
+    networks:
+      - mergin
+
+  proxy:
+    image: nginxinc/nginx-unprivileged:1.27
+    container_name: mergin-proxy-enterprise
+    restart: always
+    # run nginx as built-in user but with group mergin-family for files permissions
+    user: 101:999
+    ports:
+      - "8080:8080"
+    volumes:
+      - ./data:/data  # map data dir to host
+      - ./nginx.template:/etc/nginx/templates/default.conf.template
+    networks:
+      - mergin
+    depends_on:
+      - web
+      - server
+
+  celery-beat:
+    image: 433835555346.dkr.ecr.eu-west-1.amazonaws.com/mergin/mergin-ee-back:2025.3.0
+    container_name: mergin-celery-beat-enterprise
+    restart: always
+    user: 901:999
+    command: ["celery -A application.celery beat --loglevel=info"]
+    volumes:
+      - ./entrypoint.sh:/app/entrypoint.sh
+    env_file:
+      - .prod.env
+    depends_on:
+      - db
+      - redis
+    networks:
+      - mergin
+
+  celery-worker:
+    image: 433835555346.dkr.ecr.eu-west-1.amazonaws.com/mergin/mergin-ee-back:2025.3.0
+    container_name: mergin-celery-worker-enterprise
+    restart: always
+    user: 901:999
+    command: ["celery -A application.celery worker --pool prefork --loglevel=info"]
+    volumes:
+      - ./data:/data  # map data dir to host
+      - ./entrypoint.sh:/app/entrypoint.sh
+    env_file:
+      - .prod.env
+    depends_on:
+      - db
+      - redis
+    networks:
+      - mergin
+
+  db:
+    image: postgres:14
+    container_name: mergin-db-enterprise
+    restart: always
+    env_file:
+      - .prod.env
+    volumes:
+      - ./mergin-db-enterprise:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_DB=mergin
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres # fixme
```

Comment:
we should have some big warning somewhere to update this and also DB_PASSWORD in .prod.env

Reply:
We should remove them here and standardize everything in `.prod.env`. I mean these ones are needed for database initialization only, if user is not using an external database.